test/schema.graphql

```diff
@@ -380,6 +380,10 @@ type Query {
         COMPACT: The response includes only the value of the columns.
         """
         formatType: FormatData = COMPACT
+        """ 
+        This is an optional boolean type and is nullable
+        """
+        boolType: Boolean
```

Comment:
remove this unwanted one for now or add a better example for the same to validate

Reply:
added this to validate the changes in the tests